pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -153,6 +158,19 @@ func (plugin *kubenetNetworkPlugin) Event(name string, details map[string]interf
 	if err != nil {
 		glog.Warningf("Failed to generate CNI network config: %v", err)
 	}
+
+	configFilePath, ok := details[network.NET_PLUGIN_EVENT_POD_CIDR_CHANGE_DETAIL_PATH].(string)
+	if ok {
+		glog.V(5).Infof("Saving CNI config file at %q", configFilePath)
+		if err := os.MkdirAll(path.Dir(configFilePath), 0750); err != nil && !os.IsExist(err) {
```

Comment:
@steveeJ Good question. For now the kubelet is running as a root systemd service. This doesn't matter. But maybe we need to change to 755 so that the config file is reachable by non-root processes.